src/main/java/bitflow4j/steps/BatchPipelineStep.java

```diff
@@ -111,7 +111,7 @@ protected boolean executeIteration() throws IOException {
                         try {
                             boolean flushed = flushResults();
                             if (flushed) {
-                                logger.log(Level.INFO, "Flushed batch due to timeout (" + timeoutMs + "ms).");
+                                logger.log(Level.INFO, String.format("Flushed batch due to timeout (" + timeoutMs + "ms) for step: %s.", BatchPipelineStep.this.toString()));
```

Comment:
Since you are using String.format, please also turn the timeoutMs value into a format string parameter :)